src/benchmark/star_schema_benchmark.cpp

```diff
@@ -61,6 +61,12 @@ int main(int argc, char* argv[]) {
 
   std::cout << "- SSB scale factor is " << scale_factor << std::endl;
   context.emplace("scale_factor", scale_factor);
+  // We cannot verify the results for larger SFs since SQLite overflows integers for aggregation results. We could use
+  // dedicated result sets in these cases similar to TPC-DS. However, we need to generate these result sets using a
```

Comment:
Mention that Postgres results would be a solution but are currently not worth the effort?
Otherwise I would read that and ask: so why dont you do Postgres?

Reply:
It's not only a matter of effort. If we have to use dedicated result sets, this means we can only perform verification on the same SF the result sets were obtained. This does not solve the SF problem at all (though we could probably notice chunk pruning issues when the SF leads to more chunks per table). Gonna add a comment.